### PR TITLE
Partial FAQ update

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: Faq/faq.mixer [hunk 1]
```diff
@@ -51,7 +51,7 @@
         <a href="#2.1">2.1: Where can I download <mixer var="GAP"/></a>?
       </li>
       <li>
-        <a href="#2.2">2.2: I cannot FTP that much data. Can I get a CD?</a>
+        <a href="#2.2">2.2: I (obsolete question, removed)</a>
```
Comment (alex-konovalov) [inline Faq/faq.mixer]: First of all, `I` is part of the question, so it should be removed.

Second, I don't see any harm in keeping this question. If there will be a genuine case, we will be happy to look if we can help. I would only replace the word `FTP` by `download`.

Comment (fingolfin) [reply Faq/faq.mixer]: Of course we would be willing to help. But this is an *F*AQ, and I strongly doubt that this question is asked frequently

**Comment by ChrisJefferson** [comment]
I agree with removing this question. I'm tempted to instead just remove the question altogether (because looking through, I think quite a lot of questions need to be deleted in the same way). In the unlikely event someone linked to 2.2, they would either find the new 2.2 (if we put one in), or nothing, but web-pages change over time.

**Comment by ChrisJefferson** [comment]
But for now, I'd be happy with just nuking the question like this PR (once the extra `I` is removed), if it started the cleanup

**Comment by fingolfin** [comment]
The anchors of these question should be changed to symbolic names, and the numbered ones should at most be kept for backwards compatibility (but I'd rather get rid of them). Of course it would also be nice if most of this HTML stuff around it was generated (i.e., if the page was written in Markdown, ReST or something like that). Here we are getting close to the "let's redo the website" business, though ;-)

**Comment by alex-konovalov** [comment]
Thanks @fingolfin - this is live now.

**Pull request merged**